pkg/services/queryhistory/database.go

```diff
@@ -54,3 +54,40 @@ func (s QueryHistoryService) deleteQuery(ctx context.Context, user *models.Signe
 
 	return queryID, err
 }
+
+func (s QueryHistoryService) patchQueryComment(ctx context.Context, user *models.SignedInUser, UID string, cmd PatchQueryCommentInQueryHistoryCommand) (QueryHistoryDTO, error) {
+	var queryHistory QueryHistory
+	err := s.SQLStore.WithDbSession(ctx, func(session *sqlstore.DBSession) error {
```

Comment:
I think it's safer to wrap this and the following update query in a transaction.
```suggestion
	err := s.SQLStore.InTransaction(ctx, func(session *sqlstore.DBSession) error {
```

Reply:
Updated in https://github.com/grafana/grafana/pull/44981/commits/6ce6576df2d835488b611b989d185641dd89f052